### PR TITLE
[O11y][ActiveMQ] Add metric_type and support tooltip for period

### LINKED ISSUE #8864: [O11y][ActiveMQ] Support tooltip for period
In ActiveMQ, found that 

- Tooltip is not present for the period for each metrics data stream

Hence need to support a tooltip for the period for all metrics data streams.

## Related issue

- https://github.com/elastic/integrations/issues/6048

### LINKED ISSUE #8865: [O11y][ActiveMQ] Add metric_type for queue and broker data stream
In ActiveMQ, currently, the metric_type is not present for [queue](https://github.com/elastic/integrations/pull/6532/files#diff-f01134d018657a8a1d6215de765d426d878fa34b47387078dd3df18af083fecdL57-L62) and [broker](https://github.com/elastic/integrations/pull/6532/files#diff-05edef251f9ea072c92402735037aee611f210be6a584ac39adee29003a7710eR53-R55). Hence need to add metric_type.

## Related issue

- https://github.com/elastic/integrations/issues/6048

### PR DESCRIPTION
**Pull request by harnish-elastic**
## Proposed commit message

- Add metric_type to the fields that doesn't having it. 
- Support tooltip for period.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## Related issues

- Closes #8865 
- Closes #8864

### CODE REVIEW

**Review by milan-elastic — APPROVED** [review]
LGTM!

File: packages/activemq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.15.0
+  changes:
+    - description: Add metric_type and support tooltip for period.
+      type: bugfix
```
Comment (ritalwar) [inline packages/activemq/changelog.yml:5]: enhancement?

File: packages/activemq/data_stream/broker/manifest.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ streams:
         required: true
         show_user: true
         default: 10s
+        description: Period of fetching metrics, i.e. 1s/1m/1h.
```
Comment (ritalwar) [inline packages/activemq/data_stream/broker/manifest.yml:13]:
```suggestion
        description: The period for fetching metrics, e.g., every 1s, 1m, 1h
```

File: packages/activemq/data_stream/queue/manifest.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ streams:
         required: true
         show_user: true
         default: 10s
+        description: Period of fetching metrics, i.e. 1s/1m/1h.
```
Comment (ritalwar) [inline packages/activemq/data_stream/queue/manifest.yml:13]:
```suggestion
        description: The period for fetching metrics, e.g., every 1s, 1m, 1h
```

File: packages/activemq/data_stream/topic/manifest.yml [hunk 4]
```diff
@@ -10,6 +10,7 @@ streams:
         required: true
         show_user: true
         default: 10s
+        description: Period of fetching metrics, i.e. 1s/1m/1h.
```
Comment (ritalwar) [inline packages/activemq/data_stream/topic/manifest.yml:13]:
```suggestion
        description: The period for fetching metrics, e.g., every 1s, 1m, 1h
```

**Review by ritalwar — APPROVED** [review]

**Pull request merged**